### DISCUSSION TITLE
Recommendation; asking for pool driver

### DISCUSSION THREAD

**Discussion by spaceWrench380** [Leci37/TensorFlow-stocks-prediction-Machine-learning-RealTime#27] [General]
Hi Luis,

I would like to recommend feedback of confirmation on each step in the process in order to confirm
- the step is complete
- the files and processing completed successfully
I had a problem where I had thought everything was done, but my GPU didn't have memory needed and was crashing tensorflow. When running the final step it was failing and with all of the debugging and noise from the processing it was a challenge to look through all of the output to troubleshoot

At this point I would like to ask for the pool driver needed to progress on to step 5

**Comment by Leci37** [comment]
This file is here https://github.com/Leci37/stocks-prediction-Machine-learning-RealTime-TensorFlow/blob/master/useless/p_P_hand.zip

Be carefull some code are unupdated , did you find any problem? did you solve it ?
